### DISCUSSION TITLE
I receive ECC certificates instead of RSA

### DISCUSSION THREAD

**Discussion by Ashot-sd** [acmesh-official/acme.sh#4713] [Q&A]
When I create a certificate with the command acme.sh --issue --dns dns_myapi -d "example.com", I get an ECC certificate. If I add --keylength 2048, it works, even though it wasn't necessary to enter it. This happened after updating acme.sh, and I couldn't find any information about it in the documentation. Is this normal? Thank you.

**Comment by neilpang** [comment]
the default key type has changed from rsa to ecdsa.
If you want RSA cert, use "--keylength 2048|3072"
